packages/reporter/src/lib/shortcuts.ts

```diff
@@ -19,7 +19,26 @@ class Shortcuts {
     const isTextLike = $dom.isTextLike(event.target)
     const isAnyModifierKeyPressed = event.altKey || event.ctrlKey || event.shiftKey || event.metaKey
 
-    if (isAnyModifierKeyPressed || isTextLike) return
+    if (isTextLike) return
+
+    if (isAnyModifierKeyPressed) {
+      switch (event.key) {
```

Comment:
None of our current or previous keyboard short cuts use [`ctrl` to trigger the shortcut](https://github.com/cypress-io/cypress/commit/a80f2ba255ec51c240b4987559bbc60527f9f1c6#diff-0c07826c542debabd81f3ca774e5dfee0280f24a69aa09bb1fb4d206f5bfc4d4R24)....do we need to introduce it for this shortcut?

Reply:
I think we should. The main intention of this ticket was to restore the shortcut into the main Specs List (which is not yet implemented in this PR) - In Cy 9.x this was `Ctrl + F`, which worked since it was in the Electron app, in 10 `Ctrl + F` would step on the browser search, so `Ctrl + K` is the next best option (and matches our docs site).

Single-key shortcuts have some accessibility drawbacks (voice control-related iirc) and in the long run it would be good to move away from them.

Reply:
>  The main intention of this ticket was to restore the shortcut into the main Specs List (which is not yet implemented in this PR)

There is a shortcut for[ toggling the spec list already `f`](https://github.com/cypress-io/cypress/pull/23384/files/056df50d3d9a83180bf432d09b8d0507f5836b69#diff-0c07826c542debabd81f3ca774e5dfee0280f24a69aa09bb1fb4d206f5bfc4d4R48) ...its just a bit different since 9.x moved you back to the "launchpad" essentially.

Reply:
I will say, in 9.x it was just `f` though, which out the `ctrl+f` because a user specially added in the logic to prevent our shortcuts on `ctrl+f`: https://github.com/cypress-io/cypress/pull/20375

Reply:
What I mean is, on the equivalent of this page in Cypress 9, you could press `Cmd + F` to focus the search bar. The goal is to get a keyboard shortcut for this into Cy 10. The implications for "what should `Cmd + F` do in the runner?" are more like an after thought - since we have a specs list there as well, the same keyboard shortcut should work both places.

<img width="1428" alt="Screen Shot 2022-08-17 at 11 32 22 AM" src="https://user-images.githubusercontent.com/8340719/185180890-77cb96a3-82b1-44c3-a3cc-90d741cd943b.png">

Reply:
> I will say, in 9.x it was just f though, which out the ctrl+f because a user specially added in the logic to prevent our shortcuts on ctrl+f: #20375

Right, and I agree with that, we shouldn't step on browser search. We're talking about 2 different things though. "Go to the specs list" in 9 vs "Focus the search bar" in 9.

"Go to specs list" became "open inline specs list" and "focus the search bar" got dropped as a feature.